commit 581a1eb87cb69644de29d55fcfdd3e68b94ca1ad
Author: Ondřej Nový <ondrej.novy@firma.seznam.cz>
Date:   Mon May 9 16:03:08 2016 +0200

    Fixed unit tests on Hurd, where PATH_MAX doesn't exists

--- a/tests/fileops/cpmv_files.c
+++ b/tests/fileops/cpmv_files.c
@@ -6,6 +6,7 @@
 #include <stdlib.h> /* free() */
 #include <string.h> /* strcpy() strdup() */
 
+#include "../../src/compat/fs_limits.h"
 #include "../../src/utils/dynarray.h"
 #include "../../src/utils/fs.h"
 #include "../../src/utils/macros.h"
--- a/tests/fileops/make_dirs.c
+++ b/tests/fileops/make_dirs.c
@@ -5,6 +5,7 @@
 #include <stdlib.h> /* free() */
 #include <string.h> /* strcpy() */
 
+#include "../../src/compat/fs_limits.h"
 #include "../../src/cfg/config.h"
 #include "../../src/ui/ui.h"
 #include "../../src/utils/dynarray.h"
--- a/tests/misc/autocmds.c
+++ b/tests/misc/autocmds.c
@@ -5,6 +5,7 @@
 
 #include "../../src/cfg/config.h"
 #include "../../src/compat/os.h"
+#include "../../src/compat/fs_limits.h"
 #include "../../src/engine/autocmds.h"
 #include "../../src/utils/env.h"
 #include "../../src/utils/fs.h"
--- a/tests/misc/chase_links.c
+++ b/tests/misc/chase_links.c
@@ -8,6 +8,7 @@
 
 #include "../../src/cfg/config.h"
 #include "../../src/compat/os.h"
+#include "../../src/compat/fs_limits.h"
 #include "../../src/filelist.h"
 #include "../../src/filtering.h"
 #include "../../src/utils/dynarray.h"
--- a/tests/misc/cmdline_completion.c
+++ b/tests/misc/cmdline_completion.c
@@ -8,6 +8,7 @@
 #include <string.h>
 #include <wchar.h> /* wcsdup() */
 
+#include "../../src/compat/fs_limits.h"
 #include "../../src/cfg/config.h"
 #include "../../src/engine/abbrevs.h"
 #include "../../src/engine/cmds.h"
--- a/tests/misc/commands.c
+++ b/tests/misc/commands.c
@@ -5,6 +5,7 @@
 #include <stdio.h> /* remove() */
 #include <string.h> /* strcpy() strdup() */
 
+#include "../../src/compat/fs_limits.h"
 #include "../../src/cfg/config.h"
 #include "../../src/engine/cmds.h"
 #include "../../src/engine/functions.h"
--- a/tests/misc/flist_custom.c
+++ b/tests/misc/flist_custom.c
@@ -9,6 +9,7 @@
 #include "../../src/cfg/config.h"
 #include "../../src/cfg/info.h"
 #include "../../src/cfg/info_chars.h"
+#include "../../src/compat/fs_limits.h"
 #include "../../src/compat/os.h"
 #include "../../src/ui/fileview.h"
 #include "../../src/ui/ui.h"
--- a/tests/misc/flist_reload.c
+++ b/tests/misc/flist_reload.c
@@ -4,6 +4,7 @@
 #include <string.h> /* memset() */
 
 #include "../../src/compat/os.h"
+#include "../../src/compat/fs_limits.h"
 #include "../../src/ui/ui.h"
 #include "../../src/utils/dynarray.h"
 #include "../../src/utils/fs.h"
--- a/tests/misc/running.c
+++ b/tests/misc/running.c
@@ -6,6 +6,7 @@
 #include <stdio.h> /* snprintf() */
 #include <string.h> /* strdup() */
 
+#include "../../src/compat/fs_limits.h"
 #include "../../src/cfg/config.h"
 #include "../../src/utils/dynarray.h"
 #include "../../src/utils/fs.h"
--- a/tests/misc/sync.c
+++ b/tests/misc/sync.c
@@ -3,6 +3,7 @@
 #include <stdlib.h> /* free() */
 #include <string.h> /* strdup() */
 
+#include "../../src/compat/fs_limits.h"
 #include "../../src/cfg/config.h"
 #include "../../src/utils/dynarray.h"
 #include "../../src/cmd_core.h"
--- a/tests/utils/fswatch.c
+++ b/tests/utils/fswatch.c
@@ -2,6 +2,7 @@
 
 #include <stdio.h> /* remove() snprintf() */
 
+#include "../../src/compat/fs_limits.h"
 #include "../../src/compat/os.h"
 #include "../../src/utils/fs.h"
 #include "../../src/utils/fswatch.h"
--- a/tests/utils/parse_file_spec.c
+++ b/tests/utils/parse_file_spec.c
@@ -5,6 +5,7 @@
 #include <stdlib.h> /* free() */
 #include <string.h> /* strcmp() */
 
+#include "../../src/compat/fs_limits.h"
 #include "../../src/utils/fs.h"
 #include "../../src/utils/path.h"
 #include "../../src/utils/utils.h"
--- a/tests/utils/remove_last_path_component.c
+++ b/tests/utils/remove_last_path_component.c
@@ -1,7 +1,6 @@
 #include <stic.h>
 
-#include <limits.h> /* PATH_MAX */
-
+#include "../../src/compat/fs_limits.h"
 #include "../../src/utils/path.h"
 
 static int windows(void);
